### ISSUE TITLE
Not working for dynamic added element

### ISSUE THREAD

**Issue by samphp90** [viralpatel/jquery.shorten#28]
Hi,
I'm using your plugin with datatable and i have added some row with append method. The shorten is not working with those appended elements. Can you have any solution for it.

**Comment by jtrod** [comment]
Exactly the same is happening to me. Did you find an alternative?

**Comment by samphp90** [comment]
Yes, I write down one function and place shorten js code under that function. So, after appending the value, I'm just calling that function.

**Comment by jtrod** [comment]
Nice!!!!
